pkg/builder/container_builder.go

```diff
@@ -44,23 +44,29 @@ func ForPluginContainer(image string, pullPolicy corev1api.PullPolicy) *Containe
 }
 
 // getName returns the 'name' component of a docker
-// image (i.e. everything after the last '/' and before
-// any subsequent ':')
+// image that includes its reposiroty name, and transforms the combined
+// string into a DNS-1123 compatible name.
 func getName(image string) string {
-	slashIndex := strings.LastIndex(image, "/")
+	slashIndex := strings.Index(image, "/")
```

Comment:
If `/` isn't in the string, this will return -1 which will cause the next line to panic.

Reply:
Good catch, addressed.